src/OpenSage.Game/Data/W3d/W3dEmitterLineProperties.cs

```diff
@@ -21,7 +21,7 @@ internal static W3dEmitterLineProperties Parse(BinaryReader reader, W3dParseCont
             {
                 var result = new W3dEmitterLineProperties
                 {
-                    Flags = (W3dEmitterLineFlags)reader.ReadUInt32(),
+                    Flags = reader.ReadUInt32AsEnum<W3dEmitterLineFlags>(),
```

Comment:
On further reflection, this should be:

``` csharp
Flags = reader.ReadUInt32AsEnumFlags<W3dEmitterLineFlags>()
```

and then you can remove the new value from `W3dEmitterLineFlags`.